.vscode/launch.json

```diff
@@ -23,14 +12,30 @@
             "program": "${workspaceFolder}/cmd/porch/main.go",
             "args": [
                 "--secure-port=4443",
-                "--v=7",
-                "--standalone-debug-mode",
+                // "--v=7",
+                // "--standalone-debug-mode",
                 "--kubeconfig=${env:KUBECONFIG}",
-                "--cache-directory=${workspaceFolder}/.cache",
```

Comment:
Removing this will mean it will work into /home/<userid>/.cache, or something similar. Might be an unintended sideeffect.

Reply:
Yes, you guessed correctly :) removing this was a result of an experimentation that is not relevant to the PR. Fixing this.